sbat.c

```diff
@@ -120,4 +123,161 @@ EFI_STATUS parse_sbat(char *sbat_base, size_t sbat_size, char *buffer,
 	return efi_status;
 }
 
+EFI_STATUS
+verify_sbat(struct sbat *sbat, list_t *sbat_entries)
+{
+	unsigned int i;
+	list_t *pos = NULL, *tmp = NULL;
+
+	for (i = 0; i < sbat->size; i++) {
+		struct sbat_entry *entry = entry = sbat->entries[i];
```

Comment:
Maybe add a `verify_sbat_entry()` function or something to make this function smaller and easier to read?

Reply:
added at 158,  or did you mean to wrap even more code into that verify_single_entry ?

Reply:
That's OK, I think makes the function easier to read.